### DISCUSSION TITLE
How to extract detection signals from self-check-input event?

### DISCUSSION THREAD

**Discussion by yuanboww** [NVIDIA-NeMo/Guardrails#316] [Q&A]
Hi! my team is looking into using NeMo guardrails to run user prompts only - we want to get the signal on `whether the user message should be blocked or not` and make a decision outside of NeMo guardrail from there. Has anyone looked into that?

**Comment by drazvan** [comment]
Hi @yuanboww ! Have a look at this: https://github.com/NVIDIA/NeMo-Guardrails/tree/develop/examples/configs/guardrails_only. 
You can have a guardrails configuration that only performs the validation. In the next release we should also have a cleaner API for this.

**Reply by skamalj** [reply]
Can you link me to the config to enable this in  new version. I need this as well.
